### PR TITLE
Trade mode responsive styling

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Optimizes the display of elements depending on the responsive layout/viewport

https://github.com/cowprotocol/cowswap/assets/31534717/87083edc-9ca9-4883-9682-29d8fe6a1acc

# Todo
- Move fortune cookie and Appzi buttons to main menu on mobile/responsive

### CODE REVIEW

**Comment by fairlighteth** [comment]
@elena-zh Thanks for reviewing!

1. Had noticed it too. Perhaps I'll leave as is given the "my orders" and settings button isn't present. So there's more horizontal space available.
2. Same as 1. in this case the 'my orders' is not present so a bit more space available.
3. Yes this is intentional. When your viewport resolution width is 320px or less, I'm pushing the account to the footer (given there's no space anymore on top). The fortune cookie/appzi buttons will be taken care of.
4. Will fix
5. Yes!

**Review by alfetopito — APPROVED** [review]

**Comment by fairlighteth** [comment]
- Fixed point 4
- Re-located the Fortune widget and Appzi buttons to the mobile main menu


https://github.com/cowprotocol/cowswap/assets/31534717/816dd91a-68c7-434c-80ca-06d38e33cdf8


https://github.com/cowprotocol/cowswap/assets/31534717/1d276cd4-bbd7-4d42-aa82-e832258985eb

Screenshot 2024-03-25 at 16 18 50
Screenshot 2024-03-25 at 16 18 38

File: apps/cowswap-frontend/src/modules/fortune/containers/FortuneWidget/index.tsx [hunk 1]
```diff
@@ -288,18 +327,18 @@ const StyledCloseIcon = styled(X)`
...
   const { openFortune } = useAtomValue(fortuneStateAtom)
   const lastCheckedFortune = useAtomValue(lastCheckedFortuneAtom)
   const updateOpenFortune = useSetAtom(updateOpenFortuneAtom)
   const [isFortunesFeatureDisabled, setIsFortunesFeatureDisabled] = useAtom(isFortunesFeatureDisabledAtom)
   const openRandomFortune = useOpenRandomFortune()
   const [isNewFortuneOpen, setIsNewFortuneOpen] = useState(false)
   const [isFortunedShared, setIsFortunedShared] = useState(false)
-
-  const [today, setToday] = useState(new Date())
-  useInterval(() => setToday(new Date()), 2_000)
```
Comment (fairlighteth) [inline apps/cowswap-frontend/src/modules/fortune/containers/FortuneWidget/index.tsx:301]: @shoom3301 @alfetopito would you know why this userInterval was in place? It causes the Fortune banner to re-render every 2 seconds when opened on mobile/responsive.

Comment (alfetopito) [reply apps/cowswap-frontend/src/modules/fortune/containers/FortuneWidget/index.tsx:301]: From the code it looks like it's done to keep the date updated.
I wasn't involved so I can't say why that's important.

**Review by elena-zh — APPROVED** [review]
@fairlighteth , great job!
I wanted to report a bug with a scrollbar in the menu in Win 11 on a reduced screen, but it appears to be a known issue #1175

**Pull request merged**